examples/operations/QueueScript.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>Might be worth adding that <code>is_standard=True</code> means it looks in <code>ts_standardscripts</code> and whereas <code>is_standard=False</code> looks in ts_externalscripts. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>